x-pack/test/scalability/runner.ts

```diff
@@ -25,9 +25,14 @@ async function sendReportMetricsToTelemetry(
 ) {
   const reportRootPath = path.resolve(gatlingProjectRootPath, 'target', 'gatling');
   const fileName = path.basename(scalabilityJsonPath, path.extname(scalabilityJsonPath));
-  const journeyReportDir = fs.readdirSync(reportRootPath).filter((f) => f.startsWith(fileName));
+  const journeyReportDir = fs
+    .readdirSync(reportRootPath)
+    // Gatling report folder has unique postfix, e.g. 'api.telemetry.cluster_stats.no_cache-20230309224753010'
+    .filter((f) => f.replace(/-\d+$/, '') === fileName);
```

Comment:
```suggestion
    .filter(hasSuffix);
function hasSuffix (filePath: string): Boolean {
  const suffix: RegExp = /-\d+$/;
  return filePath.replace(suffix, '') === fileName;    
}
```

Probably should be like this per the style guide:
https://docs.elastic.dev/kibana-dev-docs/contributing/styleguide#use-descriptive-conditions